lib/core/src/Cardano/Wallet/Api.hs

```diff
@@ -481,7 +482,7 @@ type DelegationFee = "wallets"
 type PostPoolMaintenance = "stake-pools"
     :> "maintenance-actions"
     :> ReqBody '[JSON] ApiMaintenanceActionPostData
-    :> Post '[JSON] ApiMaintenanceAction
+    :> PostNoContent
```

Comment:
Need to update `swagger.yaml`

Reply:
Swagger already shows:
![Screenshot from 2020-11-12 14-41-13](https://user-images.githubusercontent.com/42900201/98947362-62e9a980-24f5-11eb-8079-ae1bdc864d6e.png)